engine/api/workflowtemplate/dao_test.go

```diff
@@ -0,0 +1,56 @@
+package workflowtemplate
+
+import (
+	"testing"
+
+	"github.com/ovh/cds/engine/api/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTemplate(t *testing.T) {
+	db := &test.SqlExecutorMock{}
+
+	expected := "SELECT * FROM workflow_template WHERE false"
+
+	_, err := Get(db, NewCriteria())
+	assert.Nil(t, err)
+	assert.Equal(t, expected, db.LastQuery().Query)
+
+	_, err = GetAll(db, NewCriteria())
+	assert.Nil(t, err)
+	assert.Equal(t, expected, db.LastQuery().Query)
+}
+
+func TestAudit(t *testing.T) {
+	db := &test.SqlExecutorMock{}
```

Comment:
nope

Reply:
answer on comment for TestAggregateAuditsOnWorkflowTemplate